stable/osg-hosted-ce/osg-hosted-ce/values.yaml

```diff
@@ -160,3 +156,9 @@ ContainerTags:
 # - Generates a test CA and self-signed host cert/key pair
 Developer:
   Enabled: false
+
+### SLATE-START ###
+SLATE:
+  Instance:
+    ID: "untagged"
+### SLATE-END ###
```

Comment:
Could you add some comments to indicate the purpose of this configuration? What do the `SLATE-START` and `SLATE-END` stanzas indicate?

Reply:
Basically a "do-not-modify" section. The SLATE API itself looks at this part of the Values, and can inject values at run-time. In particular, we want to have SLATE tag each object it creates in Kubernetes with the unique instance ID it belongs to.

Reply:
Ah ok, that's what I suspected. It'd be nice if you guys could add a layer of indirection to inject these values behind the scenes so that they don't have to be exposed to the end users

Reply:
I think the values need to appear in the chart for the base chart to work on vanilla Helm.